test/test_interpolations.jl

```diff
@@ -47,19 +47,20 @@
         # Note that not every element formulation exists for every order and dimension.
         applicable(Ferrite.getlowerorder, interpolation) && @test typeof(Ferrite.getlowerorder(interpolation)) <: Interpolation{ref_shape,func_order-1}
 
-    # Check partition of unity at random point.
-    n_basefuncs = getnbasefunctions(interpolation)
-    x = rand(Tensor{1, ref_dim})
-    f = (x) -> [shape_value(interpolation, Tensor{1, ref_dim}(x), i) for i in 1:n_basefuncs]
-    @test vec(ForwardDiff.jacobian(f, Array(x))') ≈
-        reinterpret(Float64, [shape_gradient(interpolation, x, i) for i in 1:n_basefuncs])
-    @test sum([shape_value(interpolation, x, i) for i in 1:n_basefuncs]) ≈ 1.0
-
-    # Check if the important functions are consistent
-    coords = Ferrite.reference_coordinates(interpolation)
-    @test length(coords) == n_basefuncs
-    @test shape_value(interpolation, x, n_basefuncs) == shape_value(interpolation, x, n_basefuncs)
-    @test_throws ArgumentError shape_value(interpolation, x, n_basefuncs+1)
+        n_basefuncs = getnbasefunctions(interpolation)
+        coords = Ferrite.reference_coordinates(interpolation)
+        f = (x) -> [shape_value(interpolation, Tensor{1, ref_dim}(x), i) for i in 1:n_basefuncs]
+        @testset let x = sample_random_point(ref_shape)
+            # Check partition of unity at random point.
+            @test vec(ForwardDiff.jacobian(f, Array(x))') ≈
+                reinterpret(Float64, [shape_gradient(interpolation, x, i) for i in 1:n_basefuncs])
+            @test sum([shape_value(interpolation, x, i) for i in 1:n_basefuncs]) ≈ 1.0
```

Comment:
Since this was the problem in #811, perhaps it is good to add a temporary show here in case the test fails in the future, to make it possible to debug the problem then?
```suggestion
            @test sum([shape_value(interpolation, x, i) for i in 1:n_basefuncs]) ≈ 1.0
            if sum([shape_value(interpolation, x, i) for i in 1:n_basefuncs]) !≈ 1.0
                @show x # Show coordinate in case failure, see issue #811 
            end
```